src/Resources/config/orm.xml

```diff
@@ -34,10 +36,12 @@
             <argument>%sonata.page.site.class%</argument>
             <argument type="service" id="doctrine"/>
         </service>
+        <service id="Sonata\PageBundle\Model\SiteManagerInterface" alias="sonata.page.manager.site"/>
         <service id="sonata.page.block_interactor" class="%sonata.page.block_interactor.class%" public="true">
             <argument type="service" id="doctrine"/>
             <argument type="service" id="sonata.page.manager.block"/>
         </service>
+        <service id="Sonata\PageBundle\Model\BlockInteractorInterface" alias="sonata.page.block_interactor"/>
```

Comment:
What do you think of aliasing model interfaces with the orm implementation? we can do this way or bind directly the implementation.

Reply:
@VincentLanglet . If we consider we won't implement mongodb integration anytime soon (I won't try it). This might be a good move, wdyt?

Reply:
How do we do in other bundle, like MediaBundle which supports both ORM and Mongo ?
IMHO, we should be consistent